### ISSUE TITLE
What's the status of the project?

### ISSUE THREAD

**Issue by bobylito** [christophercliff/metalsmith-webpack#25]
Hi there,

I really appreciate the work done on this project and I'm using it on one of my projects ❤️. However I'd like to know if I should continue to rely on it?

**Comment by leviwheatcroft** [comment]
Active development on v2 branch. Will publish it to npm as soon as I get collaborator credentials there.

Upgrades to v2, adds output to metalsmith meta, and caches output to avoid unnecessary builds. See it in action in [metalsmith-all-the-things](https://github.com/leviwheatcroft/metalsmith-all-the-things)

**Comment by vvo** [comment]
@leviwheatcroft Ultimately we might just publish your package on another npm name, when the author is not responsive (no hard feelings, we all have a life), that's how I do it so that your awesome work can still live somewhere.

I would fork under metalsmith-wp, open your own issues and publish.

Also, what do you think of putting webpack as a peerDep so that we can use any version 2.x? Thanks

**Comment by leviwheatcroft** [comment]
@vvo yeah, original author responded to my request for collaborator rights a while back, I've posted a separate issue #26 re collaborator rights on npm - I didn't think to ask for that at the time. We will see where we are up to with that in a week or so. My preference is to maintain this package, but I'll publish a fork if that becomes unworkable.

re: peer dependency - good idea, I posted issue #27 .. PR welcome, otherwise I'll get to it sooner or later.

**Comment by leviwheatcroft** [comment]
metalsmith-webpack is dead. long live [metalsmith-webpack-2](https://www.npmjs.com/package/metalsmith-webpack-2).

**Comment by vvo** [comment]
Awesome!

**Comment by vvo** [comment]
(you need to update your readme, it says metalsmith-webpack)

**Comment by vvo** [comment]
As a last try to takeover, maybe you could contact @christophercliff via linkedin? https://www.linkedin.com/in/christophercliff/
